package.json

```diff
@@ -2,37 +2,37 @@
   "name": "meteor-node-stubs",
   "author": "Ben Newman <ben@meteor.com>",
   "description": "Stub implementations of Node built-in modules, a la Browserify",
-  "version": "1.0.3",
+  "version": "1.0.4",
```

Comment:
Since we have major version updates here it might be a good idea to have at least minor version bump.

Reply:
@StorytellerCZ, OK, will move to `1.1.0`